src/style.css

```
@@ -0,0 +1,132 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

It is not considered the best practice to apply the reset styles to all elements using the `*` selector. It is generally recommended to apply reset styles selectively, targeting only the specific elements that need them. You can leave only box-sizing here to set this property uniformly across all elements
```suggestion
* {
  box-sizing: border-box;
}
```